src/main/groovy/edu/oregonstate/mist/recommendations/resources/RecommendationResource.groovy

```diff
@@ -0,0 +1,116 @@
+package edu.oregonstate.mist.recommendations.resources
+
+import com.google.common.base.Optional
+import edu.oregonstate.mist.api.Resource
+import edu.oregonstate.mist.recommendations.core.Recommendation
+import edu.oregonstate.mist.recommendations.core.University
+import edu.oregonstate.mist.recommendations.db.RecommendationDAO
+import org.hibernate.validator.constraints.NotEmpty
+
+import javax.ws.rs.GET
+import javax.ws.rs.Path
+import javax.ws.rs.Produces
+import javax.ws.rs.QueryParam
+import javax.ws.rs.core.MediaType
+
+/**
+ * Recommendation Resource Class
+ */
+@Path('/recommendations/')
+@Produces(MediaType.APPLICATION_JSON)
+class RecommendationResource extends Resource {
+    private final RecommendationDAO recommendationDAO
+
+    public RecommendationResource(RecommendationDAO recommendationDAO) {
+        this.recommendationDAO = recommendationDAO
+    }
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
```

Comment:
Let's add some of the metrics annotations for dropwizard: http://www.dropwizard.io/0.8.2/docs/manual/core.html#metrics